src/rust/ide/view/graph-editor/src/component/node/input/area.rs

```diff
@@ -305,6 +300,11 @@ fn select_color(styles:&StyleWatch, tp:Option<&Type>) -> color::Lcha {
 // === Area ===
 // ============
 
+/// Input ports area.
+///
+/// ## Origin
+/// Please note that the origin of the node is on its left side, centered vertically. To learn more
+/// about this design decision, please read the docs for the node.
```

Comment:
Maybe add a [doc link](https://doc.rust-lang.org/nightly/rustdoc/linking-to-items-by-name.html) to the relevant documentation here.